git/operations.go

```diff
@@ -99,6 +99,14 @@ func checkPush(ctx context.Context, workingDir, upstream, branch string) error {
 	return execGitCmd(ctx, args, gitCmdConfig{dir: workingDir})
 }
 
+func secretUnseal(ctx context.Context, workingDir string) error {
+	args := []string{"secret", "reveal", "-f"}
+	if err := execGitCmd(ctx, args, gitCmdConfig{dir: workingDir}); err != nil {
+		return errors.Wrap(err, fmt.Sprintf("git secret reveal -f"))
```

Comment:
```suggestion
		return errors.Wrap(err, "git secret reveal -f")
```

No reason to use `fmt.Sprintf` for a plain string.